manifests/profile/hathitrust/apache.pp

```diff
@@ -164,4 +164,24 @@
     }
   }
 
+  cron { 'apache connection count check':
+    command => '/usr/local/bin/ckapacheconn',
+    user    => 'root',
+    minute  => '*/15',
+  }
+
+  $http_files = lookup('nebula::http_files')
+  file { '/usr/local/bin/ckapacheconn':
+    ensure => 'present',
+    mode   => '0755',
+    source => "https://${http_files}/ae-utils/bins/ckapacheconn"
+  }
+
+  cron { 'apache restart':
+    command => '( /etc/init.d/apache2 stop; /bin/sleep 10; /etc/init.d/apache2 start ) > /dev/null',
```

Comment:
This will work, but it would be better to say `/bin/systemctl stop apache2; /bin/sleep 10; /bin/systemctl start apache2`